localdev/ches-mock/src/main.ts

```diff
@@ -0,0 +1,110 @@
+import sanitizeHtml from 'sanitize-html';
+import nodemailer from 'nodemailer';
+import express, { Request, Response } from 'express';
+
+const app = express();
+app.use(express.json());
+const port = 3025;
+
+interface Email {
+  from?: string;
+  subject: string;
+  body: string;
+  to: string[];
+  cc?: string[];
+  bcc?: string[];
+  attachments?: {
+    content: string | Buffer;
+    filename: string;
+    contentType?: string;
+    encoding?: 'base64' | 'binary' | 'hex';
+  }[];
+}
+
+// Email sending function
+export const sendViaMailPit = async (email: Email): Promise<void> => {
+  try {
+    const transporter = nodemailer.createTransport({
+      host: 'mailpit',
+      port: 1025,
+      secure: false,
+      auth:
+        process.env.MAIL_USERNAME && process.env.MAIL_PASSWORD
```

Comment:
is the codebase tested on the local environment? if then how the environment variables starting with `MAIL_` defined?

Reply:
it was tested, since it password and username are not required(can be undefined) it worked well locally - changed